packages/spor-accordion-react-native/src/utils.tsx

```diff
@@ -0,0 +1,71 @@
+import { useCallback, useEffect, useRef } from "react";
+import { Animated, LayoutChangeEvent } from "react-native";
+
+export function useExpandAnimation(
+  isExpanded: boolean,
+  defaultExpanded?: boolean
+) {
+  const hasRenderedRef = useRef(false);
+  const { currentBounds, setBounds } = useMutableBounds();
+  const heightAnim = useRef(new Animated.Value(0)).current;
+  const opacityAnim = useRef(new Animated.Value(0)).current;
+
+  const handleLayoutChanges = useCallback(
+    (event: LayoutChangeEvent) => {
+      if (!hasRenderedRef.current && defaultExpanded) {
+        heightAnim.setValue(event.nativeEvent.layout.height);
+        opacityAnim.setValue(1);
+      }
+      hasRenderedRef.current = true;
+      setBounds(event);
+    },
+    [heightAnim, opacityAnim, hasRenderedRef, setBounds, defaultExpanded]
+  );
+
+  useEffect(() => {
+    if (currentBounds.height === 0) {
+      return;
+    }
+    const heightAnimation = Animated.spring(heightAnim, {
+      toValue: isExpanded ? currentBounds.height : 0,
+      useNativeDriver: false,
+      velocity: 1,
+    });
+    const opacityAnimation = Animated.timing(opacityAnim, {
+      toValue: isExpanded ? 1 : 0,
+      useNativeDriver: false,
```

Comment:
Hvorfor bruker man ikke `useNativeDriver` her?

Reply:
Godt spørsmål. Det ser ut til å fungere å kombinere to animasjoner med useNativeDriver true og false i salgsappen, men i rn-demo appen henger animasjonen seg opp etterhvert når jeg prøver på det.

Reply:
Aha! Jeg kan ikke nok om animasjoner i RN til å kunne uttale meg kjenner jeg – men om det er potensielt problematisk å ha det satt til `true` så er det jo bare å droppe :) Men i teorien så skal det gå fortere/smoothere om man bruker `useNativeDriver`, korrekt?